### PR TITLE
Learning Pool contribution bundle 12th March 2015

### PR DESCRIPTION
**Pull request by brian-learningpool**
Fixes for the following tickets:
AB-292 - Extensions against 'config' do not seem to save
AB-160 - Add poster images to projects
AB-302 - Inform user that required field is missing when adding new asset
AB-316 - Search does not work
AB-314 - Double scroll bar
AB-334 - Video asset thumbnails should be animated gifs
AB-318 - Display issue with tiles when there are many characters used for
the Course title

Also includes the following:
- Minor IE 9 fixes
- Added support for client-side configs
- Corrected 'Back to courses' button
- Added missing language strings
- Fixes around drag and drop, copy and paste
- Role permissions synced on a server restart
- Fix for asset manager filters
- General UI fixes
- Proper support for shared courses
- Poster images now available on courses
- Progress indicator on preview
- Re-factoring of build process
- Fixes with theme and menu selection to persist selection after new
  versions are installed
- Added support for _trackingId values
- Install updated
- Context menu issues fixed

Contributors:
Brian Quinn
Daryl Hedley
Dennis Heaney
Finbar Tracey

### CODE REVIEW

**Comment by darylhedley** [comment]
+1 - works pretty nicely now

**Comment by dennis-learningpool** [comment]
+1

File: frontend/src/core/user/templates/login.hbs [hunk 1]
```diff
@@ -12,8 +12,7 @@
         <div class="login-new-user col-1-2">
 
             <div class="login-new-user-inner">
-                 Welcome to the Adapt Builder.<br/><br />
-                 This is a pre-release version only intended for selected audiences. 
+                 Welcome to Learning Pool's Adapt Builder<br/><br />
```
Comment (taylortom) [inline frontend/src/core/user/templates/login.hbs]: Need to change this

File: plugins/content/subscribed/index.js [hunk 2]
```diff
@@ -0,0 +1,561 @@
+// LICENCE https://github.com/adaptlearning/adapt_authoring/blob/master/LICENSE
+/**
+ * Shared courses in the Learning Pool Styleee
```
Comment (taylortom) [inline plugins/content/subscribed/index.js]: Should be changed

File: routes/index/index.hbs [hunk 3]
```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <title>Adapt Origin</title>
+    <title>Adapt Builder</title>
```
Comment (taylortom) [inline routes/index/index.hbs]: Do we have an agreed name for the tool?

File: routes/lang/en.json [hunk 4]
```diff
@@ -1,5 +1,5 @@
 {
-  "app.productname": "Origin",
+  "app.productname": "Adapt Builder",
```
Comment (taylortom) [inline routes/lang/en.json]: Do we have an agreed name for the tool?

File: routes/lang/en.json [hunk 5]
```diff
@@ -65,7 +75,7 @@
   "app.resetpassword" : "Reset password",
   "app.submit" : "Submit",
   "app.newpassword" : "New password",
-  "app.forgotpasswordblurb": "Enter the e-mail address associated with your Adapt account, then click Continue. We'll email you a link to a page where you can easily create a new password.",
+  "app.forgotpasswordblurb": "Enter the e-mail address associated with your Learning Pool Adapt account, then click Continue. We'll email you a link to a page where you can easily create a new password.",
```
Comment (taylortom) [inline routes/lang/en.json]: Should be changed

File: routes/lang/en.json [hunk 6]
```diff
@@ -200,7 +216,7 @@
   "app.databasepass": "Database Password",
   "app.databaseport": "Database Port",
   "app.emailadminpasswordresetsubject": "Your Adapt password has been reset",
-  "app.emailadminpasswordresetbody": "Your password for Adapt has been reset.  Your new password is: %{data}",
+  "app.emailadminpasswordresetbody": "Your password for Learning Pool's Adapt Builder has been reset.  Your new password is: %{data}",
```
Comment (taylortom) [inline routes/lang/en.json]: Should be changed

File: plugins/content/subscribed/package.json [hunk 7]
```diff
@@ -0,0 +1,4 @@
+{
+  "displayName": "Learning Pool shared courses",
```
Comment (taylortom) [inline plugins/content/subscribed/package.json]: Should be changed

**Comment by taylortom** [comment]
All looks good to me, just those few references to Learning Pool that need changing :)

Also think we should open up the debate around the name of the tool - of the two used at the moment (origin and builder), I'm more in favour of origin. Maybe this is something we could open up to the community?

**Comment by brian-learningpool** [comment]
Tom, I've removed the references to Learning Pool as per your comments. 

I agree that we should open up a debate about the product name on the community.  I always thought of 'Origin' as more of a build name myself.

**Comment by taylortom** [comment]
+1, looks good Brian :)

**Pull request merged**